internal/auth/repository/eventsourcing/eventstore/key.go

```diff
@@ -2,27 +2,58 @@ package eventstore
 
 import (
 	"context"
+	"os"
 	"time"
 
+	"github.com/caos/logging"
 	"gopkg.in/square/go-jose.v2"
 
 	"github.com/caos/zitadel/internal/auth/repository/eventsourcing/view"
+	"github.com/caos/zitadel/internal/crypto"
+	"github.com/caos/zitadel/internal/errors"
+	"github.com/caos/zitadel/internal/eventstore/spooler"
+	"github.com/caos/zitadel/internal/eventstore/v2"
+	"github.com/caos/zitadel/internal/id"
+	"github.com/caos/zitadel/internal/key/model"
+	key_view "github.com/caos/zitadel/internal/key/repository/view"
+	"github.com/caos/zitadel/internal/v2/command"
 )
 
 type KeyRepository struct {
-	View               *view.View
-	SigningKeyRotation time.Duration
+	Commands                 *command.CommandSide
+	Eventstore               *eventstore.Eventstore
```

Comment:
Do you use eventstore v2 on purpose?

Reply:
which else should i use? v1 will be removed / deprecated when new-eventstore will be merged into master